common-pipeline/operators/operator-v3-schema.json

```diff
@@ -125,7 +125,12 @@
           ]
         }
       },
-      "oneOf": [
+      "anyOf": [
```

Comment:
parameters that has `id` and `type` or `id` and `enum` were matching more than one schema after the third case was added, so this needed to be changed to `anyOf` since `oneOf` requires the instance to match exactly one

Reply:
The test harness selectschema example doesnt require `type` or `enum` https://github.com/elyra-ai/canvas/blob/master/canvas_modules/harness/test_resources/parameterDefs/selectschema_paramDef.json

Reply:
It probably should have `type` added to the example since it would be string values.